docs/README.md

```diff
@@ -32,6 +32,79 @@ $ ./build_docs.sh
 
 The site can be viewed at http://localhost:1313/
 
+## Include externally hosted documentation
+
+With the ongoing efforts to move Flink's connectors from this repository to individual, dedicated
+repositories, this also requires the documentation to be hosted outside this repo. However, 
+we still want to serve all documentation as a whole on the Flink documentation website.
+
+In order to achieve this, we're using [Hugo Modules.](https://gohugo.io/hugo-modules/configuration/) 
+to create a virtual filesystem. 
+
+Adding new externally hosted documentation requires the following steps to be taken:
+
+1. (If necessary) Move the existing documentation to the new repository
+2. In this new repository, in the `docs` folder, create a file `go.mod` containing:
+
+```go
+module github.com/apache/flink-connector-<repositoryname>/docs
+
+go 1.18
+```
+
+Replace <repositoryname> with the name of your repository.
+See https://github.com/apache/flink-connector-elasticsearch/tree/main/docs/go.mod for an example.
+3. In this new repository, in the `docs` folder, create a `config.toml` file containing:
+
+```yaml
+module:
+  mounts:
+    - source: content/docs/connectors/datastream/<filename>.md
+      target: content/docs/connectors/datastream/<filename>.md
+      lang: en
+    - source: content/docs/connectors/table/<filename>.md
+      target: content/docs/connectors/table/<filename>.md
```

Comment:
I'm just gonna assume we can't just set both source/target to `content` because it would ignore all the Flink-local content?

Reply:
I'll double check if it's not possible, this is more because this is the explicit method. I think it could be possible, because in one of my first attempts I didn't yet remove the Elasticsearch documentation from this repo. However, Hugo didn't have any issue in building it, it just used the local one first.